### PR TITLE
Ignore timeout issue when polling logs

### PR DESCRIPTION
**Pull request by timqian**
- [x] Ignore errors when polling logs with `sls logs -t`
- [x] Remove unnecessary code blocks

Related ticket
- https://app.asana.com/0/1200011502754281/1200601499823399

### CODE REVIEW

**Review by zongUMR — APPROVED** [review]

File: src/cli/commands-cn/logs.js [hunk 1]
```diff
@@ -156,7 +140,13 @@ module.exports = async (config, cli, command) => {
 
     cli.sessionStart('监听中');
     setInterval(async () => {
-      const newLogList = await getLogList();
+      let newLogList;
+      // ignore timeout issue when polling new logs
+      try {
+        newLogList = await getLogList();
+      } catch (error) {
```
Comment (zongUMR) [inline src/cli/commands-cn/logs.js:147]: Can't we fix this error, or just ignore it?

Comment (timqian) [reply src/cli/commands-cn/logs.js:147]: As you may already know, there is a small chance that users will meet timeout issues when requesting backend APIs due to network issues. We cannot do much on it. And for `sls logs -t`, by default, we will poll `getLogs` API every 2 seconds, if some of the requests timeout, we should not show the unhandled promise rejection in users console. Ignoring them should be the best way.

Comment (zongUMR) [reply src/cli/commands-cn/logs.js:147]: A question: If there is a non-timeout error occur here, but we just `return 0`, and can not catch those kinds of error anymore?

Comment (timqian) [reply src/cli/commands-cn/logs.js:147]: When users execute `sls logs -t`, we will do a `getLogList` request first and if there are any "non-timeout" issues, we should have been able to catch the issue.

**Pull request merged**